package.json

```diff
@@ -1,11 +1,20 @@
 {
   "name": "@yext/answers-headless",
   "version": "1.2.0",
-  "description": "",
+  "description": "A library for powering UI components for Yext Answers integrations",
   "author": "slapshot@yext.com",
   "license": "ISC",
   "main": "./dist/commonjs/src/index.js",
   "module": "./dist/esm/src/index.js",
+  "keywords": [
+    "search",
+    "nlp",
+    "ai",
+    "answers",
+    "yext",
+    "react",
+    "components"
```

Comment:
Did product ask for "components" and "react" here? This library doesn't have any components in it, and also doesn't use react

Reply:
I would probably put "redux" here if anything

Reply:
Product only gave me one list, but none of the repositories have keywords in package.json. I thought I'd just put the same list, but I was wondering if we need different lists of keywords for all of the repositories?

Reply:
I merged this branch by mistake, but I reverted the changes...

Reply:
I merged this into main by mistake, but I've reverted the changes.